frontend/js/createTournament.js

```diff
@@ -0,0 +1,26 @@
+const form = document.getElementById("tournament-form");
+
+console.log("Loaded createTournament.js");
+
+form.addEventListener("submit", function (event) {
+    event.preventDefault();
+    const formValues = {
+        name: form.name.value,
+        player_code: form.player_code.value,
+        admin_code: form.admin_code.value
+    };
+
+    fetch('http://localhost:8000/tournaments', {
+        method: 'post',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(formValues)
+    }).then(function(res) {
```

Comment:
Please refactor these `then` calls to `await` statements. That improves the readbility.

Reply:
Done